tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -219,10 +159,10 @@ private function lockFileUsingFileId(
 		$responseXml = $this->featureContext->getResponseXml($response, __METHOD__);
 		$xmlPart = $responseXml->xpath("//d:locktoken/d:href");
 		if (isset($xmlPart[0])) {
-			$this->tokenOfLastLock[$user][$filePath] = (string) $xmlPart[0];
+			$this->tokenOfLastLock[$user] = (string) $xmlPart[0];
```

Comment:
@saw-jan I'll solve storing lock token when I combine lockFileInSpace() and lockFile() and move tests from coreApiWebdavLocks to apiSpaces